integration-tests/spring-web/pom.xml

```diff
@@ -19,6 +19,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-spring-web</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-undertow</artifactId>
+        </dependency>
```

Comment:
So my only comment here is that we have no idea if things work without undertow being there. Or do we?

Reply:
We do because of 2 things:. 

1) I added one if the internal tests
2) I also tested the Spring Web quickstart

Reply:
Ah yes, the added test is in the deployment module. I missed that, sorry.

Looks good to me then.